azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/localuser/LocalParticipantViewModel.kt

```diff
@@ -117,7 +118,8 @@ internal class LocalParticipantViewModel(
         val displayVideo = shouldDisplayVideo(videoStreamID)
         val displayLobbyOverlay = shouldDisplayLobbyOverlay(callingState)
         val displayFullScreenAvatar =
-            shouldDisplayFullScreenAvatar(displayVideo, displayLobbyOverlay, viewMode)
+            shouldDisplayFullScreenAvatar(displayVideo, displayLobbyOverlay, viewMode) &&
```

Comment:
Same here `callingState == CallingStatus.CONNECTED` should be part of the `shouldDisplayFullScreenAvatar`